src/Util.h

```diff
@@ -44,11 +44,6 @@
 #define HALIDE_NO_USER_CODE_INLINE HALIDE_NEVER_INLINE
 #endif
 
-// On windows, Halide needs a larger stack than the default MSVC provides
-#ifdef _MSC_VER
-#pragma comment(linker, "/STACK:8388608,1048576")
-#endif
-
```

Comment:
Where did `1048576` go? It's the default reservation size, so this might cause extra increases to the stack size. Will that have a meaningful impact on start-up times?

Reply:
I see that it would be more consistent to port the commit size to cmake, too. After reading the description of the commit size on https://docs.microsoft.com/en-us/cpp/build/reference/stack?view=msvc-160 I did not feel confident that this needs to be set, but I forgot to bring it up in the discussion.

Reply:
I wrote this code originally. The thinking was that 1MB is probably the smallest useful stack size, and 8MB is what we frequently need, so may as well allocate things in 1MB chunks. Given that the OS needs to reserve the full 8MB of address space anyway, and lazy allocation of physical memory for pages is the norm now, I doubt the second arg even does anything.

Reply:
Good to know. Dropping it is probably not an issue, then.